test/test-sourcemapper.ts

```diff
@@ -69,7 +69,7 @@ function testTool(
         ' it states that it has mapping info for files it knows about',
       done => {
         assert.strictEqual(
-          sourcemapper.hasMappingInfo(inputFilePath),
+          sourcemapper.getMapInfoInput(inputFilePath) !== null,
```

Comment:
nit, we could use `assert.notStrictEqual(sourcemapper.getMapInfoInput(inputFilePath), null, ...)`

Reply:
Done.